inventory

```diff
@@ -0,0 +1,25 @@
+[prometheus-servers]
+10.0.1.145
+10.0.1.225
+10.0.1.240
```

Comment:
@Miyurz Can we avoid IPs hard-coding ? Instead we can fetch it using cloud provider API or tags.

Reply:
good idea! this was not in the initial PoC so can you raise an issue and add work on it ?

Reply:
@RahulMahale  https://github.com/Miyurz/telemetry/issues/4